tests/stdlib/tstropt.nim

```diff
@@ -0,0 +1,74 @@
+discard """
+  targets: "c cpp js"
+"""
+
+import std/stropt
+
+
+proc teststrip() =
+  var a = "  vhellov   "
```

Comment:
this really needs a helper proc/template to transform each block of 5 lines into a single line; much easier to read/write/edit; this works:

```nim
const whitespaces = {' ', '\t', '\v', '\r', '\l', '\f'}
proc strip2(a: string, leading = true, trailing = true, chars: set[char] = whitespaces): string =
  result = a
  strip(result, leading, trailing, chars)

proc teststrip() =
  doAssert "  vhellov   ".strip2 == "vhellov"
  doAssert "  vhellov   ".strip2(leading = false) == "  vhellov"
  doAssert "  vhellov   ".strip2(trailing = false) == "vhellov   "
  ...
```